integration/common/openlineage/common/provider/dbt.py

```diff
@@ -633,9 +633,21 @@ def extract_namespace(self, profile: Dict) -> str:
             return "bigquery"
         elif profile['type'] == 'redshift':
             return f"redshift://{profile['host']}:{profile['port']}"
+        elif profile['type'] == 'spark':
+            if profile['method'] in ['thrift', 'odbc', 'http']:
```

Comment:
_Minor_: Can we define the method types as enums?

Reply:
Done.